utils/constants.tsx

```diff
@@ -662,8 +656,8 @@ export const DefaultRolePermissions = {
         Permissions.MANAGE_PRIVATE_CHANNEL_MEMBERS,
         Permissions.DELETE_POST,
         Permissions.EDIT_POST,
-        Permissions.MANAGE_CREATE_EMOJIS,
-        Permissions.MANAGE_DELETE_EMOJIS,
+        Permissions.CREATE_EMOJIS,
+        Permissions.DELETE_EMOJIS,
```

Comment:
Great catch! I don't know where these are actually surfaced in the UI, but I imagine you probably saved someone a headache somewhere down the line.